cinch/roles/dockerize/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+- name: download dockerize
+  get_url:
+    dest: /tmp/dockerize.tar.gz
+    url: https://github.com/jwilder/dockerize/releases/download/{{ dockerize_version }}/dockerize-{{ dockerize_architecture }}-{{ dockerize_version }}.tar.gz
+  register: dockerize_download
+
+- name: unarchive dockerize
+  unarchive:
+    src: /tmp/dockerize.tar.gz
+    remote_src: true
+    dest: /usr/local/bin
+  when: "{{ dockerize_download | changed }}"
+
+- name: make sure dockerize has proper attributes
+  file:
+    dest: /usr/local/bin/dockerize
+    mode: 0755
+    owner: root
+    group: root
+    state: file
```

Comment:
No, it runs solely inside of a Docker container. The "dockerize" utility allows inserting environment variables into a file and templating it out. For machine-based installs, we template values directly into the config files with Ansible, but with Docker image builds we need to allow the user to supply those values at run time instead of at configure time.